mecsimcalc/general_utils.py

```diff
@@ -13,38 +13,41 @@ def input_to_file(
         metadata: bool = False
     ) -> Union[io.BytesIO, Tuple[io.BytesIO, str]]
 
-    Transforms a Base64 encoded string into a file object. Optionally, the file metadata can also be returned.
+    Transforms a Base64 encoded string into a file object. Optionally, returns the file metadata.
 
-    # Parameters
+    Parameters
+    ----------
     input_file : str
-        A Base64 encoded string prefixed with metadata.
+        A Base64 encoded string prefixed with metadata, indicating the type and encoding of the file.
     metadata : bool, optional
-        If set to True, the function also returns the metadata. Default is False.
-
-    # Raises
-    * `ValueError`:
-        If the input string does not contain ';base64,' which is required to separate metadata and file data.
-
-    # Returns
-    * `Union[io.BytesIO, Tuple[io.BytesIO, str]]` :
-        * `metadata | False` : io.BytesIO - Returns a file object containing the file data.
-        * `metadata | True` : Tuple[io.BytesIO, str] - Returns a tuple containing the file object and the metadata.
-
-    **Note:** The file object is open and can be used with Python file functions (e.g., file.read()).
-
-
-    # Examples
-    **Without metadata**
-    >>> input_file = inputs["input_file"]
-    >>> file = msc.input_to_file(input_file)
-
-    (file is now ready to be used with Python file functions) (e.g., file.read())
-
-    **With metadata**
-    >>> input_file = inputs["input_file"]
-    >>> file, metadata = msc.input_to_file(input_file, metadata=True)
-
-    (metadata holds information about the file, such as the file type)
+        If True, the function also returns the metadata extracted from the input string. Defaults to False.
+
+    Returns
+    -------
+    Union[io.BytesIO, Tuple[io.BytesIO, str]]
+        If `metadata` is False, returns an `io.BytesIO` object containing the decoded file data.
+        If `metadata` is True, returns a tuple containing the `io.BytesIO` object and a string representing the metadata.
+
+    Raises
+    ------
+    ValueError
+        If the input string does not contain ';base64,', which is required to separate metadata from the file data.
+
+    Notes
+    -----
+    The returned file object is open and can be used with Python file functions, such as `read()`.
+
+    Examples
+    --------
+    Without metadata:
+    >>> input_file = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUA..."
```

Comment:
The reason we had ` input_file = inputs["input_file"] ` is because this is how users will store the data on mecsimcalc

Although this might be the actual data being stored, but nobody will input it this way
`input_file = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUA..."`

Reply:
Please fix this for all other functions as well